doc/api/configuration.md

```diff
@@ -105,10 +105,12 @@ Automatically limit the size of the returned result sets (e.g., add a `LIMIT` at
 
 ## `displaylimit`
 
-Default: `None` (no limit)
+Default: `60`
```

Comment:
Gotcha

Reply:
yeah, I comment on this here: https://github.com/ploomber/jupysql/issues/419#issuecomment-1522456875

pandas shows 10, although the setting has a default value of 60 (?) - either it's a bug or there's something about this config that I don't understand.

anyway, I propose the following. default = 10

if 0 or None, display all (just like pandas)

Reply:
I think displaying 10 rows as a default makes sense, but currently, if the user will specify a limit in the query (that is larger than 10) - it might be confusing.

e.g:

```
%sql select * from "penguins.csv" limit 20
```

Will return 10 rows as default.

So in order to fetch 20 rows, he needs to set the `displaylimit` to None. 

Suggestion:
If the query contains `limit X` and the X > `displaylimit` - we can print a message for the user suggesting to update the configuration.

Reply:
Detect containing limit keyword:
I think this needs to be reconsidered since:

1. MS SQL doesn't natively support LIMIT keyword. Instead, they use something like OFFSET X ROWS FETCH NEXT X ROWS ONLY. Ref https://stackoverflow.com/questions/603724/how-to-implement-limit-with-sql-server
2. The query %sql select * from "penguins.csv" limit 20 for me is almost equivalent to %sql select * from "penguins.csv", since they mean The length of query result is large than `displaylimit`.

I think if we detect the query statement might be over-engineering this

Reply:
@tonykploomber I see your point.

I'm still not sure it's intuitive for the user to modify the default configuration to fetch more rows and I guess the most common case here is to run different queries and explore the results. Maybe we can just add a message at the bottom suggesting changing the default configuration (if the number of rows from the query is larger than displaylimit).

Reply:
I agree with this! We should tell user to checkout configuration part in our doc if they want to see more